src/cljs/rems/administration/create_catalogue_item.cljs

```diff
@@ -15,10 +16,20 @@
             [rems.util :refer [navigate! fetch post! put! trim-when-string]]))
 
 (defn- update-loading [db]
-  (let [progress (::loading-progress db)]
-    (if (<= 2 progress)
-      (dissoc db ::loading-progress ::loading?)
-      (assoc db ::loading-progress (inc progress)))))
+  (let [progress (::loading-progress db)
+        catalogue-item-id (::catalogue-item-id db)]
+    (cond
+      (<= progress 1)
```

Comment:
instead of counting progress numbers, you could do something like

```
(cond
  (not (and (::resources db) (::workflows db) (::forms db)))
  {} ;; do nothing

  ...
```

Reply:
Good idea, done.